sphinx/ext/todo.py

```diff
@@ -212,17 +219,17 @@ def merge_info(app, env, docnames, other):
 def visit_todo_node(self, node):
     # type: (nodes.NodeVisitor, todo_node) -> None
     self.visit_admonition(node)
-    # self.visit_admonition(node, 'todo')
 
 
 def depart_todo_node(self, node):
-    # type: (nodes.NodeVisitor, todo_node) -> None
+    # type: (HTMLTranslator, todo_node) -> None
     self.depart_admonition(node)
 
 
 def latex_visit_todo_node(self, node):
-    # type: (nodes.NodeVisitor, todo_node) -> None
-    title = node.pop(0).astext().translate(tex_escape_map)
+    # type: (LaTeXTranslator, todo_node) -> None
+    title_node = cast(nodes.title, node.pop(0))
```

Comment:
This is also ugly. This goes pop out and casting at same time. I worried it would confuse somebody.